docs/content/docs/dev/datastream/scala_api_extensions.md

```diff
@@ -198,14 +198,8 @@ To use this extension exclusively, you can add the following `import`:
 import org.apache.flink.api.scala.extensions.acceptPartialFunctions
 {{< /highlight >}}
 
-for the DataSet extensions and
-
-{{< highlight scala >}}
-import org.apache.flink.streaming.api.scala.extensions.acceptPartialFunctions
-{{< /highlight >}}
-
 The following snippet shows a minimal example of how to use these extension
-methods together (with the DataSet API):
```

Comment:
It's probably out of the scope of this PR. I wonder if we should simply also remove all Scala API documentations.

Reply:
Sure. I saw there is another work item [[FLIP-265] Remove all Scala APIs](https://issues.apache.org/jira/browse/FLINK-29741) in 2.0 burndown, maybe we can track in that thread.